app/views/warehouse_reports/outcomes/es_report/_index.haml

```diff
@@ -0,0 +1,24 @@
+- title = "Emergency Shelter Dashboard"
+- content_for :title, title
+%h1= title
+= render '/warehouse_reports/breadcrumbs', tab: 'operational'
+= render 'warehouse_reports/outcomes/shared/form'
+= render 'warehouse_reports/outcomes/es_report/overview'
+= render 'warehouse_reports/outcomes/es_report/time_in_program'
+= render 'warehouse_reports/outcomes/es_report/program_makeup'
+= render 'warehouse_reports/outcomes/es_report/outcomes'
+= render 'warehouse_reports/outcomes/es_report/returns'
+= render 'warehouse_reports/outcomes/shared/computation_description'
+- content_for :page_js do
+  :javascript
+    var stabilization_chart_data = #{@report.time_in_stabilization_data.to_json.html_safe}
+    var timeInStabilizationChart = new App.WarehouseReports.Rrh.Time('.jTimeInStabilizationChart', stabilization_chart_data, '#{clients_warehouse_reports_rrh_index_path(passed_params.merge(metric: :time_in_stabilization))}');
+    var destination_chart_data = #{@report.destinations.to_json.html_safe}
+    var destination_chart = new App.WarehouseReports.Rrh.Destination('.rrh__destination-chart', '.rrh__destination-chart-legend', destination_chart_data, '#{clients_warehouse_reports_rrh_index_path(passed_params.merge(metric: :destination))}');
+    var ph_returns_data = #{@report.ph_returns_for_chart.to_json.html_safe}
+    var ph_returns_chart = new App.WarehouseReports.Rrh.Returns('#jPhReturnsChart', ph_returns_data, '#{clients_warehouse_reports_rrh_index_path(passed_params.merge(metric: :return_after_exit_to_ph))}');
+    var returns_data = #{@report.returns_for_chart.to_json.html_safe}
+    var returns_chart = new App.WarehouseReports.Rrh.Returns('#jReturnsChart', returns_data, '#{clients_warehouse_reports_rrh_index_path(passed_params.merge(metric: :return_after_exit_to_any))}');
+    var percent_exiting_stabilization_data = #{@report.percent_exiting_stabilization_data.to_json.html_safe}
+    var percent_exiting_stabilization_chart = new App.WarehouseReports.Rrh.Makeup('.percentExitingStabilizationChart', percent_exiting_stabilization_data, '#{clients_warehouse_reports_rrh_index_path(passed_params.merge(metric: :percent_exiting_stabilization))}')
```

Comment:
The paths in here and for TH, maybe PSH?  All need adjustments, they are loading the `RRH` data.